### PR TITLE
Pin redis version to 2.10.6

### PR DESCRIPTION
**Pull request by drgarcia1986**
To avoid some trouble with Redis [3.x series](https://github.com/andymccurdy/redis-py#upgrading-from-redis-py-2x-to-30)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     install_requires=[
         "Django>=1.11",
-        "redis>=2.10.0",
+        "redis==2.10.6",
```
Comment (jdufresne) [inline setup.py]: I think a range like `<3.0.0` would be more accurate and usable.

Comment (drgarcia1986) [reply setup.py]: LGTM 👍 
Done ✅

**Comment by michaeljonaitis** [comment]
Yas please

**Comment by jdufresne** [comment]
An alternative is PR #344 which adds support for newer redis-py and is backwards compatible with older versions.

**Comment by Maks3w** [comment]
Or both. Do a minor release for this and feature release for the compatibility PR

**Comment by niwinz** [comment]
Released a intermediary version with this change (4.9.1)
Thanks.

**Pull request closed without merging**